src/FramesMonitor.js

```diff
@@ -70,14 +71,19 @@ class FramesMonitor extends EventEmitter {
             throw new Errors.ConfigError('exitProcessGuardTimeoutInMs param should be a positive integer.');
         }
 
+        if (analyzeDurationInMs !== undefined && (!_.isSafeInteger(analyzeDurationInMs) || analyzeDurationInMs <= 0)) {
+            throw new Errors.ConfigError('You should provide a correct analyze duration, bastard.');
```

Comment:
Don't use rude lexical terms, please =)

Btw, I propose to not force a user to set this option. We may set it by default to 5 sec? As I remember, it's a default value for the ffmpeg.

Reply:
Oops, 10 lines below I see the code that sets this default value.